.github/contributing.md

```diff
@@ -0,0 +1,24 @@
+# How to properly make a request in Minehut Meta
+
+So, you want to suggest a feature, report a bug, or maybe suggest a change for the filter? Always remember to follow these guidelines: 
+
+1. Use the issue templates provided
```

Comment:
got all of these from the readme directly, but now it will show when making an issue